### PR TITLE
add type=wallet to ons

### PR DESCRIPTION
**Pull request by darcys22**
Will allow for the adding of wallet mappings following the merge of:
https://github.com/oxen-io/oxen-core/pull/1403

### CODE REVIEW

**Comment by darcys22** [comment]
https://oxen.rocks/darcys22/loki-core/1402-ons-names-for-transfers/oxen-electron-wallet-1.5.6-a0383784-mac.dmg

File: src/components/lns/lns_input_form.vue [hunk 1]
```diff
@@ -333,7 +344,10 @@ export default {
       const submitRecord = {
         ...this.record,
         name: this.record.name.toLowerCase(),
-        value: this.record.value.toLowerCase()
+        value:
+          this.record.type === "wallet"
+            ? this.record.value
```
Comment (kylezs) [inline src/components/lns/lns_input_form.vue:349]: so wallet names are case sensitive?

File: src/i18n/en-us.js [hunk 2]
```diff
@@ -428,6 +430,7 @@ export default {
     signature: "Signature to verify",
     transactionNotes: "Additional notes to locally attach to the transaction",
     unsignedData: "The data as it should look before it was signed",
+    walletAddress: "wallet address to map LNS name to",
```
Comment (kylezs) [inline src/i18n/en-us.js:433]:
```suggestion
    walletAddress: "Wallet address LNS name will map to",
```

File: src/components/lns/lns_input_form.vue (hunk 1 above)
Comment (darcys22) [reply src/components/lns/lns_input_form.vue:349]: yeah base58

**Pull request merged**